src/Centreon/Infrastructure/MonitoringServer/MonitoringServerRepositoryRDB.php

```diff
@@ -100,20 +102,44 @@ public function findServers(): array
             'is_activate' => 'ns_activate'
         ]);
 
-        $request = $this->translateDbName(
-            'SELECT SQL_CALC_FOUND_ROWS * FROM `:db`.nagios_server'
-        );
-
         // Search
         $searchRequest = $this->sqlRequestTranslator->translateSearchParameterToSql();
-        $request .= !is_null($searchRequest) ? $searchRequest : '';
 
         // Sort
         $sortRequest = $this->sqlRequestTranslator->translateSortParameterToSql();
-        $request .= !is_null($sortRequest) ? $sortRequest : ' ORDER BY id DESC';
 
         // Pagination
-        $request .= $this->sqlRequestTranslator->translatePaginationToSql();
+        $paginationRequest = $this->sqlRequestTranslator->translatePaginationToSql();
+
+        return $this->findServers($searchRequest, $sortRequest, $paginationRequest);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function findServersWithoutRequestParameters(): array
+    {
+        return $this->findServers(null, null, null);
+    }
+
+    /**
+     * Find servers.
+     *
+     * @param string|null $searchRequest Search request
+     * @param string|null $sortRequest Sort request
+     * @param string|null $paginationRequest Pagination request
+     * @return MonitoringServer[]
+     * @throws \Exception
+     */
+    private function findServers(?string $searchRequest, ?string $sortRequest, ?string $paginationRequest): array
+    {
+        $request = $this->translateDbName(
+            'SELECT SQL_CALC_FOUND_ROWS * FROM `:db`.nagios_server'
+        );
+
+        $request .= !is_null($searchRequest) ? $searchRequest : '';
+        $request .= !is_null($sortRequest) ? $sortRequest : ' ORDER BY id DESC';
+        $request .= !is_null($paginationRequest) ? $paginationRequest : '';
```

Comment:
```suggestion
        $request .= !is_null($searchRequest) ? $searchRequest : '';
        $request .= !is_null($sortRequest) ? $sortRequest : ' ORDER BY id DESC';
        $request .= !is_null($paginationRequest) ? $paginationRequest : '';
```
$request .= $searchRequest ?? '';
$request .= $sortRequest ?? 'ORDER BY id DESC';
$request .= $paginationRequest ?? '';